lib/grpc/server.ex

```diff
@@ -454,6 +454,7 @@ defmodule GRPC.Server do
 
       iex> GRPC.Server.send_reply(stream, reply)
   """
+  @deprecated "Use GRPC.Stream API instead"
```

Comment:
```suggestion
  @deprecated "Use GRPC.Stream API instead", since: "0.12.0"
```

Reply:
In fact, the streams API covers all types of RPCs in the gRPC protocol in a unified way. Therefore, we don't need this for making unary requests. However, I've reverted this deprecation for now.